libraries/AP_LandingGear/AP_LandingGear.h

```diff
@@ -94,8 +94,8 @@ class AP_LandingGear {
     AP_Int8     _pin_deployed_polarity;
     AP_Int8     _pin_weight_on_wheels;
     AP_Int8     _pin_weight_on_wheels_polarity;
-    AP_Int16    _deploy_alt;
-    AP_Int16    _retract_alt;
+    AP_Float    _deploy_alt_m;
```

Comment:
I don't think we can change the type like this.  I think you can just leave it as AP_Int16 though

Reply:
Yep, over stepped. I will revert it.